mobile_wallet/src/lib.rs

```diff
@@ -349,6 +325,98 @@ fn sign_message_aux(input: &str) -> anyhow::Result<String> {
     ))?)
 }
 
+#[derive(common::SerdeDeserialize)]
+struct MicroCCDAmount {
+    #[serde(rename = "microGtuAmount")]
+    amount: u64,
+}
+
+#[derive(common::SerdeDeserialize)]
+struct AccountAddressJSON {
+    address: String,
+}
```

Comment:
We also have `AccountAddress` defined in `contracts_common` with the right serialziation.